static/app/views/organizationGroupDetails/eventToolbar.tsx

```diff
@@ -238,4 +250,11 @@ const DescriptionList = styled('dl')`
   margin: 0;
 `;
 
+const NavigationContainer = styled('div')`
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  column-gap: ${space(1)};
```

Comment:
I think we just use `gap` in all the places. No other examples of `column-gap` in the repo.
It might need `flex-direction` in order to make that work, i havn't checked it tho.

Reply:
gap is just shorthand for row-gap column-gap but I'll update them